magento-research/pwa-studio
# bug: fix pdp image jutter

## Description

See issue

## Related Issue





Closes #1476 .

## Verification Steps

For minicart:

1. Sign in, add an item to cart and open minicart. The image should appear.
2. Clear app cache and disable cache. Refresh the page.
3. Open the cart, you should see the item in cart but the image should be a grey box.

For PDP:
1. Check that images do not cause jutter on the PDP when changing image via clicking thumbnails.

## Screenshots / Screen Captures (if appropriate)

## Proposed Labels for Change Type/Package




## Checklist:
